build.gradle

```diff
@@ -34,4 +34,11 @@ dependencies {
 
     compileOnly 'org.jenkins-ci.plugins:credentials:2.1.10@jar'
     compileOnly 'org.jenkins-ci.plugins:plain-credentials:1.0@jar'
+
+    testCompile 'org.mockito:mockito-core:2.+'
```

Comment:
Is there a reason we are using an older version of mockito? And why are we using a range for the version?

Reply:
The versions that Steve used for Polaris Jenkins are:
    testCompile group: 'org.powermock', name: 'powermock-core', version: '2.0.2'
    testCompile group: 'org.powermock', name: 'powermock-reflect', version: '2.0.2'
    testCompile group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.2'
    testCompile group: 'org.powermock', name: 'powermock-module-junit4', version: '2.0.2'
    testCompile group: 'org.mockito', name: 'mockito-core', version: '2.23.4'

We can probably use those exact same versions here for whichever Mockito libraries we need

Reply:
Though I believe we should aim for using JUnit5 over JUnit 4 wherever possible.